tools/clang/test/CodeGenSPIRV/intrinsics.vkcrossdevicescope.hlsl

```diff
@@ -0,0 +1,21 @@
+// Run: %dxc -T vs_6_0 -E main
+
+struct SInstanceData {
+  float4x3 VisualToWorld;
+  float4 Output;
+};
+
+struct VS_INPUT	{
+  float3 Position : POSITION;
+  SInstanceData	InstanceData : TEXCOORD4;
+};
+
+float4 main(const VS_INPUT v) : SV_Position {
+	const SInstanceData	I = v.InstanceData;
+
+// CHECK: OpStore %scope %uint_0
+  uint32_t scope = vk::CrossDeviceScope;
```

Comment:
I realize this goes through spir-v generation which I'm less familiar with, but I'm surprised this isn't optimized away since it doesn't contribute to the output.

Reply:
For the purposes of FileCheck tests, by default we disable optimizations to get exactly what the backend generated. Adding `-O3` to the first line (`// Run: %dxc ....`) enables optimizations.